Dockerfile

```diff
@@ -2,19 +2,26 @@
 FROM python:3.7-alpine
 
 ARG BUILD_DIR=/build
+ARG MOUNT_SOURCE_CODE=/src/taskboot
 
 # Add img
 RUN apk add --no-cache img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 
 # Setup other deps
 RUN apk add --no-cache git skopeo docker
 
-# Install taskboot from mounted source code
 # Use a dedicated build dir to avoid building in bind mount
-RUN --mount=type=bind,target=/src/taskboot \
-  cd /src/taskboot && \
-  mkdir -p ${BUILD_DIR} && \
-  pip install --no-cache-dir --build ${BUILD_DIR} . && \
-  rm -rf ${BUILD_DIR}
+RUN mkdir -p ${BUILD_DIR}
```

Comment:
Run this mkdir directly in the last command, the one with pip, so at the end of the build this directory will not be in the image.